tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -84,12 +87,13 @@ static bool ShouldPartBeIncludedInPDB(UINT32 FourCC) {
   switch (FourCC) {
   case hlsl::DFCC_ShaderDebugName:
   case hlsl::DFCC_ShaderHash:
+  case hlsl::DFCC_CompilerVersion:
```

Comment:
Should this also include `DFCC_ShaderSourceInfo`?

Reply:
This function just checks if a part from the DXIL container should be copied over to the PDB container. The `DFCC_ShaderSourceInfo` is added separately. I'll rename this function to make its purpose clearer.